endpoints/v2/manifest.py

```diff
@@ -345,6 +337,7 @@ def delete_manifest_by_digest(namespace_name, repo_name, manifest_ref):
 
 
 def _write_manifest_and_log(namespace_name, repo_name, tag_name, manifest_impl):
+    _validate_schema1_manifest(namespace_name, repo_name, manifest_impl)
```

Comment:
When I started working on saving the proxied manifest to the db the code was doing it right on the view function. To make that work I needed to separate validation logic from the creation logic because validation [of schema1 manifests] should happen using the upstream registry's namespace, while creation of the manifest should use Quay's namespace.

While the separation is no longer needed it might be worth keeping it as it clarifies the flow.